minecraft_servers/eindino/manifest.json

```diff
@@ -0,0 +1,42 @@
+{
+  "server_name": "eindino",
+  "nice_name": "einDino",
+  "direct_ip": "eindino.net",
+  "server_wildcards": [
+    "eindino.de",
+    "eindino.net"
+  ],
+  "supported_languages": [
+    "de", "en"
+  ],
+  "social": {
+    "web": "https://forum.eindino.net",
+    "web_shop": "https://shop.eindino.net",
+    "web_support": "https://forum.eindino.net",
+    "twitter": "einDinoNET",
+    "discord": "https://discord.eindino.net",
```

Comment:
Custom links are reserved for partners only; use a normal discord.gg/[INVITE] link instead